test/unit/settings_test.py

```diff
@@ -140,10 +144,12 @@ class TestEIA860Settings:
     def test_860m(self):
         """Test validation error is raised when eia860m date is within 860 years."""
         settings_cls = Eia860Settings
+        original_eia80m_date = settings_cls.eia860m_date
         settings_cls.eia860m_date = "2019-11"
 
         with pytest.raises(ValidationError):
             settings_cls(eia860m=True)
+        settings_cls.eia860m_date = original_eia80m_date
```

Comment:
this test was changing the `eia860m_date` in the class itself, not an instantiated object, so it was breaking my tests below that were using an 860 settings object. i'm honestly a lil surprised this hasn't bitten anyone else in the butt. so i just reset the date back to the original after the test. But maybe there is probably a better way to do this?

Reply:
Oh wild. So you would run the tests and the class variable for future ETL runs would be incorrect?

Reply:
yes exactly! It was a little wild. If I moved my lil tests that use `Eia860Settings` to the top of the module it would work, but if they were run after this `test_860m` it would give me the validation error that this test was trying to get.

I'm not sure how long this change persisted... but I assume we should either be very careful about editing these settings classes like this or find a work around it. But for now I figured the reset to the original was sufficent.

Reply:
I guess another option would be to subclass the Eia860Settings and change the class variable.